Gemfile.lock

```diff
@@ -15,19 +20,26 @@ GEM
     pry (0.13.1)
       coderay (~> 1.1)
       method_source (~> 1.0)
+    public_suffix (4.0.5)
     rack (2.2.3)
     rack-protection (2.0.8.1)
       rack
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rake (13.0.1)
     ruby2_keywords (0.0.2)
+    safe_yaml (1.0.5)
     sinatra (2.0.8.1)
       mustermann (~> 1.0)
       rack (~> 2.0)
       rack-protection (= 2.0.8.1)
       tilt (~> 2.0)
     tilt (2.0.10)
+    vcr (6.0.0)
+    webmock (3.8.3)
```

Comment:
Do you think this is why the build is failing? Maybe it is not allowing requests cause these are still in here

Reply:
Answer was yes, I just ran bundle install and everything is passing now.